tests/linalg_test.py

```diff
@@ -1305,6 +1305,18 @@ def testDiagonal(self, shape, dtype, offset):
     self._CheckAgainstNumpy(np_fun, lax_fun, args_maker)
     self._CompileAndCheck(lax_fun, args_maker)
 
+  def testTrace(self):
```

Comment:
Here I added a simple test instead of running a large set of test case for example compared to above testDiagonal test. The reason is that `jnp.linalg.trace` is a thin wrapper over jnp.trace which should be already extensively tested, so IMO there is no need to run large set of test cases.